examples/next-with-app-router/package.json

```
@@ -0,0 +1,28 @@
+{
+  "name": "next-with-app-router",
```

```suggestion
  "name": "@almacareer/spirit-example-next-with-app-router",
```

I suggest using `@almacareer` scope and also adding `spirit-example-` prefix. The workspace can be filtred by the name then.